### DISCUSSION TITLE
Things to address on Turing podcast on ASG

### DISCUSSION THREAD

**Discussion by RaoOfPhysics** [alan-turing-institute/asg-research-communications#77] [Ideas]
Hi folks. @mooniean will be interviewing me for the Turing podcast on 31 August, where I’ll be talking, mainly, about ASG.

Could you please let me know if there are any key points you would like me to address?

I’m going to brush up on https://www.turing.ac.uk/research/asg of course!

**Comment by mooniean** [comment]
I usually prepare a few questions/discussion points before hand, I can also adapt them if there's specific things you'd like to talk about :)

**Comment by ZaynabIsmail** [comment]
I'm impact and end of programme report obsessed so perhaps some points on what has been achieved by ASG which has benefited society, how it has been a mobilising force for ECR careers and how powerful and dynamic the CTPs have been.  We can discuss further if that helps?
